test/api.jl

```diff
@@ -79,9 +79,10 @@ addparam!(rs, k1, disablechecks = true)
 @test numreactionparams(rs) == 3
 
 rnmat = @reaction_network begin
+@parameters α β
```

Comment:
I thought that now 
```julia
rnmat = @reaction_network begin
    α, S + 2I --> 2I
    β, 3I --> 2R + S
end
```
should work too? Why is the `@parameters` line needed?

Reply:
I had a couple of 100 of these to change. In almost all cases adding the @parameters thing shouldn't matter, but I didn't want to look closer on all, and still risk messing up. I just went through all and replaced with a similar @paramters option.

I am adding additional tests to show that not having these (which should be what most people do) also work. But yes, had I written the tests from scratch I would not have had the `@parameters` part.